Fix installation doc comment ungenerated application bundle

Neodork opened the pull request:
Pr for https://github.com/sonata-project/SonataNewsBundle/issues/272

OskarStark commented:
Thank you, but please add a step in the doc where you have to uncomment this line explicitly :+1:

Neodork commented:
Updated the installation doc, added some Symfony documentation standards. 

Is there a general template we should work towards? If you look at the documentation for the notification bundle you can clearly see the difference. 

Left the composer.json instructions in after deleting them because some people might want to copy+paste and update instead of just use the commands.

Neodork commented:
Ping @OskarStark

OskarStark commented:
@Neodork can you please squash your commits and rebase?

Thank you

core23 commented:
Ping @Neodork

Neodork commented:
Yes my bad, I will finish it up before monday, needs a refactor and the commit message is lacking. Been busy.

Neodork commented:
Ping @core23

OskarStark commented:
@Neodork can you please rebase your PR?

Neodork commented:
Working on this next monday, think it needs an enter at the end of the json block.

Neodork commented:
Ping @OskarStark anything on the error Travis is giving me?

greg0ire commented on Resources/doc/reference/installation.rst:
```diff
@@ -1,12 +1,25 @@
 
     {
         "require": {
+            // ...
```
This is not valid JSON

greg0ire commented:
> Ping @OskarStark anything on the error Travis is giving me?

See my comment

Neodork commented:
ping @greg0ire Had to remove the // ... but the documentation might be confusing without it. Any opinions?

greg0ire commented:
Maybe recommend using the CLI instead, since it will not affect existing packages? So `composer require my/package:~42.7 my/other-package:~24.3 you/get-it:~1.0.0`

Neodork commented:
Ping @greg0ire This should be oke, removed the install by pasting into composer.json.

greg0ire commented:
LGTM

OskarStark commented on Resources/doc/reference/installation.rst:
```diff
@@ -24,9 +21,9 @@ if you use the API.
 
 .. code-block:: php
 
-    <?php
```
please keep this, otherwise its not rendered correctly in some cases

OskarStark commented on Resources/doc/reference/installation.rst:
```diff
@@ -152,13 +159,14 @@ if you use the API.
-.. code-block:: php
+* Enable the application bundles:
 
-    <?php
```
please keep this

OskarStark commented on Resources/doc/reference/installation.rst:
```diff
@@ -167,11 +175,29 @@ if you use the API.
+        orm:
+            entity_managers:
+                default:
+                    #metadata_cache_driver: apc
```
please remove the cache stuff, not needed here

Neodork commented:
ping @OskarStark

OskarStark commented:
Thank you @Neodork
